views/js/pciCreator/ims/likertScaleInteraction/runtime/js/renderer.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016-2024 (original work) Open Assessment Technologies SA;
+ *
+ */
+define([
+    'likertScaleInteraction/runtime/js/assets',
+    'text!likertScaleInteraction/runtime/css/likertScaleInteraction.css'
+], function(assets, css) {
+    'use strict';
+
+    function renderStyle(container) {
+        const rootElt = container.querySelector('.likertScaleInteraction');
+        let styleTag = rootElt.querySelector('style');
+
+        if (!styleTag) {
+            styleTag = document.createElement('style');
+            styleTag.innerHTML = css;
+            rootElt.prepend(styleTag);
+        }
+    }
```

Comment:
**question:** Why is this? Usually, the stylesheet can be managed by the loader...

Reply:
Removed.